### PR TITLE
Add Abigail Afi Gbadago as organizer

### PR DESCRIPTION
**Pull request by AfiMaameDufie**
## Description

* Took out the md file because it was in the wrong location and moved it to the right folder.

### CODE REVIEW

File: _organizers/abigail-afi-gbadago.md [hunk 1]
```diff
@@ -3,6 +3,6 @@ github: AfiMaameDufie
 layout: base
 name: Abigail Afi Gbadago
 photo_url: /static/img/organizers/afi.jpeg
-role: Website Team, Communications Team
+role: Programs, Website
```
Comment (drewbrew) [inline _organizers/abigail-afi-gbadago.md]:
```suggestion
role: Program Committee, Website Committee
```

Let's make it a bit more formal perhaps?

Comment (AfiMaameDufie) [reply _organizers/abigail-afi-gbadago.md]: Thank you, @drewbrew. 
Done.

File: _organizers/abigail-afi-gbadago.md [hunk 2]
```diff
@@ -3,6 +3,6 @@ github: AfiMaameDufie
 layout: base
 name: Abigail Afi Gbadago
 photo_url: /static/img/organizers/afi.jpeg
-role: Program Committee, Website Committee
+role: Programs Committee, Website Committee
```
Comment (drewbrew) [inline _organizers/abigail-afi-gbadago.md]:
```suggestion
role: Program Committee, Website Committee
```

Just keeping it in line with the official name of the team

**Comment by AfiMaameDufie** [comment]
@drewbrew All good now?

**Review by drewbrew — APPROVED** [review]

**Pull request merged**